site/docs/v1/tech/core-concepts/tenants.adoc

```diff
@@ -53,6 +54,44 @@ The named issuer used to sign tokens.  Typically a fully-qualified domain name.
 [field]#Login Theme# [optional]#Optional#::
 The Theme associated with this Tenant; determines which templates to render for interactive work-flows.
 
+===== Form Settings
+
+[.api]
+[field]#Admin user form# [optional]#Optional#::
+The form that will be used in the FusionAuth UI for adding and editing users.
++
+Updating this field requires a paid edition of FusionAuth.
+
+==== Connectors
+
+Connectors can be enabled on a per tenant basis with a Connector policy.
+
+image::connectors/tenant-connector-tab.png[The Tenant Connector policy configuration tab.,width=1200,role=shadowed]
+
+Full documentation on link:/docs/v1/tech/connectors/[Connectors] and link:/docs/v1/tech/connectors/[Connector Policies] can be found link:/docs/v1/tech/connectors/[here].
+
+== Add Connector Policy Dialog
+
+If you click on the `Add policy` button on this page you will be presented with the following dialog.
+
+[.dialog-screenshot]
+--
+=====
+image::tenant-configuration-connector-add-policy.png[Add Connector Policy,width=300,role=box-shadow]
+=====
+--
+
+=== Form Fields
+
+[.api]
+[field]#Connector# [required]#Required#::
+The Connector to be used for this policy.
+
+[field]#Domains# [optional]#Optional#::
```

Comment:
This field has a default if not specified. See https://fusionauth.io/docs/v1/tech/apis/tenants/#create-a-tenant